lib/html_parser.dart

```diff
@@ -286,6 +286,7 @@ class HtmlRichTextParser extends StatelessWidget {
     });
 
     return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
```

Comment:
Can we make this a parameter on the `Html` widget that defaults to `CrossAxisAlignment.center` so as to make this a non-breaking change?